### PR TITLE
Fix pressed button bug

### PR DESCRIPTION
**Pull request by pengin7384**
- 이전 문제에서 눌린 버튼은 key를 -1로 주도록 설정하여서 press효과가 사라지도록 구현하였습니다

### CODE REVIEW

File: src/components/ui/QuestionBoard.js [hunk 1]
```diff
@@ -189,10 +198,11 @@ class QuestionBoard extends React.Component {
                 <ButtonWrapper>
                     {questions[count].Answers.map((answer, i) => {
                         if (answer) {
+                            const key = this.getButtonKey(i);
                             return (
-                                <AnswerButton key={`${i}`}
-                                    onClick={() => this.onButtonClick(i,questions[count].Id)}>
-                                        {questions[count].Answers[i]}
+                                <AnswerButton key={key}
+                                    onClick={() => this.onButtonClick(i, questions[count].Id, key)}>
```
Comment (minsulee2) [inline src/components/ui/QuestionBoard.js:204]: Coding convention
Please make onClick moving underneath the key

File: src/components/ui/QuestionBoard.js [hunk 2]
```diff
@@ -139,18 +140,26 @@ class QuestionBoard extends React.Component {
...
         if (this.state.count < this.props.questions.length - 1) {
-            this.setState({ count: this.state.count + 1 });
+            this.setState({ count: this.state.count + 1, prevButtonKey: key });
         } else {
             this.context.toggleRedirect();
             this.setRedirect();
         }
     }
 
+    getButtonKey = (index) => {
```
Comment (minsulee2) [inline src/components/ui/QuestionBoard.js:155]: Can it be tied up? like if() else if() else() and this.state.prevButtonKey s on the same side.

File: src/components/ui/QuestionBoard.js (hunk 1 above)
Comment (pengin7384) [reply src/components/ui/QuestionBoard.js:204]: Done!

File: src/components/ui/QuestionBoard.js (hunk 2 above)
Comment (pengin7384) [reply src/components/ui/QuestionBoard.js:155]: Done! getButtonKey was removed

**Pull request merged**